_templates/sidebar_links.html

```diff
@@ -75,7 +71,17 @@
     href="https://github.com/ipython/ipython/wiki/Dev:-Index">Development information</a></li>
     <li><a class="reference external" href="http://travis-ci.org/#!/ipython/ipython">
         Travis CI</a></li>
-  </ul>          
+  </ul>
+</div>
+<div class="sidebarblock">
```

Comment:
Ok, but then we should get a decision. Remove gitter? or not? Remove reddit ? or not? ( I wasn't even aware it existed). The wiki is not of real help AFAICT either.